03 Writing Algorithms/14 Datasets/02 QuantConnect/00.json

```diff
@@ -1 +1,13 @@
-{"type": "landing", "heading": "QuantConnect", "subHeading": "", "content": "<p>\nQuantConnect was founded in 2012 to serve quants everywhere with the best possible algorithmic trading technology. Seeking to disrupt a notoriously closed-source industry, QuantConnect takes a radically open-source approach to algorithmic trading. Through the QuantConnect web platform, more than 50,000 quants are served every month.\n</p>", "alsoLinks": [], "featureShortDescription": {"03": "", "01": "", "02": "", "04": ""}}
\ No newline at end of file
+{
+    "type": "landing", 
+    "heading": "QuantConnect", 
+    "subHeading": "", 
+    "content": "<p>\nQuantConnect was founded in 2012 to serve quants everywhere with the best possible algorithmic trading technology. Seeking to disrupt a notoriously closed-source industry, QuantConnect takes a radically open-source approach to algorithmic trading. Through the QuantConnect web platform, more than 50,000 quants are served every month.\n</p>", 
+    "alsoLinks": [], 
+    "featureShortDescription": {
+        "03": "Splits, devidends, delistings, symbol change events of US Equities", 
+        "01": "Select US Equities by price, volume, and dollar volume", 
+        "02": "Select constituent US Equities from ETFs", 
+        "04": "Handling continuous Futures"
+    }
+}
```

Comment:
I believe that this landing page and Tradier's are code generated, so these changes will be reverted by the code generator.

Reply:
yah. Updated the `alternative-dataset-code-gen` and tested for avoid erasing `00.json`